rgw/v2/tests/nfs_ganesha/test_on_s3_io.py

```diff
@@ -35,208 +46,203 @@ def test_exec(rgw_user_info_file, config):
     io_info_initialize.initialize(basic_io_structure.initial())
     write_user_info = AddUserInfo()
 
-    try:
-        test_info.started_info()
-        with open(rgw_user_info_yaml, "r") as f:
-            rgw_user_info = yaml.safe_load(f)
-        mount_point = rgw_user_info["nfs_mnt_point"]
-        nfs_ganesha = PrepNFSGanesha(rgw_user_info_file=rgw_user_info_file)
-        mounted = nfs_ganesha.initialize(write_io_info=False)
-        if mounted is False:
-            raise TestExecError("mount failed")
-        if (
-            nfs_ganesha.rgw_user_info["nfs_version"] == 4
-            and nfs_ganesha.rgw_user_info["Pseudo"] is not None
-        ):
-            log.info("nfs version: 4")
-            log.info("adding Pseudo path to writable mount point")
-            mount_point = os.path.join(mount_point, nfs_ganesha.rgw_user_info["Pseudo"])
-            log.info("writable mount point with Pseudo: %s" % mount_point)
-        log.info("authenticating rgw user")
-
-        # authenticate
-        auth = Auth(rgw_user_info)
-        rgw_conn = auth.do_auth()
-        # add user_info io_info yaml file
-        user_info_add = basic_io_structure.user(**rgw_user_info)
-        write_user_info.add_user_info(user_info_add)
-        if config.io_op_config.get("create", None) is True:
-            # create buckets
-            for bc in range(config.bucket_count):
-                bucket_name_to_create = utils.gen_bucket_name_from_userid(
-                    rgw_user_info["user_id"], rand_no=bc
+    test_info.started_info()
+    with open(rgw_user_info_yaml, "r") as f:
+        rgw_user_info = yaml.safe_load(f)
+    mount_point = rgw_user_info["nfs_mnt_point"]
+    nfs_ganesha = PrepNFSGanesha(rgw_user_info_file=rgw_user_info_file)
+    mounted = nfs_ganesha.initialize(write_io_info=False)
+    if not mounted:
+        raise TestExecError("mount failed")
+    if (
+        nfs_ganesha.rgw_user_info["nfs_version"] == 4
+        and nfs_ganesha.rgw_user_info["Pseudo"] is not None
+    ):
+        log.info("nfs version: 4")
+        log.info("adding Pseudo path to writable mount point")
+        mount_point = os.path.join(mount_point, nfs_ganesha.rgw_user_info["Pseudo"])
+        log.info("writable mount point with Pseudo: %s" % mount_point)
+    log.info("authenticating rgw user")
+
+    # authenticate
+    auth = Auth(rgw_user_info)
```

Comment:
Could we pass the SSL? That is `auth = Auth(rgw_user_info, ssl=config.ssl)`

Reply:
Hi @psathyan yep I have passed SSL configuration. Apparently deletion tests were taking time during the same but increasing the sleep time from `time.sleep(200)` to `time.sleep(250)` helps out and tests are passing.

Reply:
supported in s3, 
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/lib/s3/auth.py#L32
not in swift
https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/lib/swift/auth.py#L22-L30